### PR TITLE
NFDIV-4210: Update hub page when respondent represented and international address

### PR DESCRIPTION
**Pull request by FaisalMoJ**
### Change description ###

Update hub page when respondent represented and international address

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-4210

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

**Review by DelythJustice — APPROVED** [review]

**Review by DelythJustice — APPROVED** [review]

**Review by DelythJustice — APPROVED** [review]

File: src/main/steps/common/progress-bar.njk [hunk 1]
```diff
@@ -9,14 +9,14 @@
     label: {
       html: textWithVisuallyHiddenSpan(submitted, displayState.isAtOrAfter('Submitted'))
     },
-    complete: displayState.isAtOrAfter('Submitted'),
-    active: displayState.at('AwaitingDocuments')
+    complete: displayState.isAtOrAfter('Submitted') or displayState.at('AwaitingHWFDecision'),
+    active: displayState.at('AwaitingDocuments') and not (displayState.isAtOrAfter('Submitted') or displayState.at('AwaitingHWFDecision'))
```
Comment (adamg-hmcts) [inline src/main/steps/common/progress-bar.njk:13]: Just checking - do we need the second condition after "and not" to determine whether it is "active", as we check for a specific match against AwaitingDocuments for the first part of the condition?

File: src/main/steps/common/progress-bar.njk [hunk 2]
```diff
@@ -35,14 +35,14 @@
...
```
Comment (adamg-hmcts) [inline src/main/steps/common/progress-bar.njk:39]: Same question as [line 13](https://github.com/hmcts/nfdiv-frontend/pull/3884/files#r1768142405) - not sure we need the second part of the active condition?

File: src/main/steps/common/progress-bar.njk [hunk 3]
```diff
@@ -35,14 +35,14 @@
...
   },  {
     label: {
       html: textWithVisuallyHiddenSpan(courtChecks, displayState.isBefore('Submitted'))
     },
     complete: displayState.isAfter('Submitted'),
-    active: displayState.isAtOrAfter('AwaitingAos')
+    active: displayState.isAfter('AwaitingAos') or displayState.at('AwaitingHWFDecision')
```
Comment (adamg-hmcts) [inline src/main/steps/common/progress-bar.njk:45]: Should we still be using `isAtOrAfter('AwaitingAos')` rather than `isAfter('AwaitingAos')`?

Comment (FaisalMoJ) [reply src/main/steps/common/progress-bar.njk:45]: Yes that's true we need isAtOrAfter 👍

File: src/main/steps/common/progress-bar.njk (hunk 1 above)
Comment (FaisalMoJ) [reply src/main/steps/common/progress-bar.njk:13]: Discussed this on Slack and we will leave it as is

**Review by DelythJustice — APPROVED** [review]

**Pull request merged**